### DISCUSSION TITLE
Interacting with app while phone is locked

### DISCUSSION THREAD

**Discussion by ghost** [BimmerGestalt/AAIdrive#405] [Q&A]
Is it right that browsing in spotify via idrive5 is only possible when de app is open? Because when i lock my phone, i cant skip songs etc.

**Comment by hufman** [comment]
That is not intended, the app is meant to be used entirely from the car while the phone is put away! Does your phone have aggressive battery saving modes, perhaps? It sounds like the OS could be suspending the app while the screen is off and preventing it from responding to input from the car. I have plans in #403 to detect such problems and to hint to the user to change the battery saving options.

**Reply by ghost** [reply]
Normally i do not use the battery saving mode especially in the car because it is mostly loading. I also did not see any settings of restrictions..i've put the spotify app on the list of apps which should never go into standby and will test it as soon as possible..

Further i'll check my phonesettings in detail..

**Reply by hufman** [reply]
Along with Spotify, AAIdrive itself needs to be put on the special list of apps that can run in the background, and probably even MyBMW. Every phone has different battery-saving options, which makes it frustrating to provide a good consistent experience.

**Reply by ghost** [reply]
Finally i had time to test it...still the same behavior :(

**Reply by ghost** [reply]
do i have to connect in every case via USB to get de max. out of the app? i normally just connect via Bluetooth, no USB at all.. if i can provide  you any further information to investigate for this problem, just tell me

**Reply by hufman** [reply]
Thank you very much for your patience! You'll actually have more functionality when you use the app over Bluetooth, because several audio features aren't available over USB.
